.gitignore

```diff
@@ -109,3 +109,6 @@ venv.bak/
 .mypy_cache/
 .dmypy.json
 dmypy.json
+
+# IDEs
```

Comment:
Please use a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) instead of polluting this project's gitignore

Reply:
You are right